src/app/.env.dev

```diff
@@ -5,4 +5,5 @@ DAD_JOKES_BASE_URL='https://www.reddit.com/r/dadjokes'
 APP_STORE_URL_IOS=https://apps.apple.com/us/app/uno-calculator/id1464736591
 APP_STORE_URL_Android=https://play.google.com/store/apps/details?id=uno.platform.calculator
 REMOTE_CONFIG_FETCH_INTERVAL_MINUTES=1
-DIAGNOSTIC_ENABLED=true
\ No newline at end of file
+DIAGNOSTIC_ENABLED=true
+IS_DATA_OBSCURE=true
```

Comment:
I think this should be obscure**d** to stay consistent with `BugseeConfigurationData`.

Reply:
This parameter is the initial value but it can be updated from the Bugsee diagnostic overlay later (in generated apps only this value will be taken into account)